modules/serverless-kafka-sink-security-config.adoc

```diff
@@ -0,0 +1,103 @@
+
+.Prerequisites
+
+* The {ServerlessOperatorName}, Knative Eventing, and the `KnativeKafka` custom resources (CRs) are installed on your {product-title} cluster.
+* Kafka sink is enabled in the `KnativeKafka` CR.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+* You have a Kafka cluster CA certificate stored as a `.pem` file.
+* You have a Kafka cluster client certificate and a key stored as `.pem` files.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Create the certificate files as a secret in the same namespace as your `KafkaSink` object:
++
+[IMPORTANT]
+====
+Certificates and keys must be in PEM format.
+====
+
+** For authentication using SASL without encryption:
++
+[source,terminal]
+----
+$ oc create secret -n <namespace> generic <secret_name> \
+  --from-literal=protocol=SASL_PLAINTEXT \
+  --from-literal=sasl.mechanism=<sasl_mechanism> \
+  --from-literal=user=<username> \
+  --from-literal=password=<password>
+----
+
+** For authentication using SASL and encryption using TLS:
++
+[source,terminal]
+----
+$ oc create secret -n <namespace> generic <secret_name> \
+  --from-literal=protocol=SASL_SSL \
+  --from-literal=sasl.mechanism=<sasl_mechanism> \
+  --from-file=ca.crt=caroot.pem \ <1>
+  --from-literal=user=<username> \
+  --from-literal=password=<password>
+----
+<1> The `ca.crt` can be omitted to use the system's root CA set. For example, if you are using a public cloud managed Kafka service such as Red Hat OpenShift Streams for Apache Kafka.
+
+** For encryption using TLS without client authentication:
++
+[source,terminal]
+----
+$ oc create secret -n <namespace> generic <secret_name> \
+  --from-literal=protocol=SSL \
+  --from-file=ca.crt=<my_caroot.pem_file_path> \ <1>
+  --from-literal=user.skip=true
+----
+<1> The `ca.crt` can be omitted to use the system's root CA set. For example, if you are using a public cloud managed Kafka service such as Red Hat OpenShift Streams for Apache Kafka.
+
+** For authentication and encryption using TLS:
++
+[source,terminal]
+----
+$ oc create secret -n <namespace> generic <secret_name> \
+  --from-literal=protocol=SSL \
+  --from-file=ca.crt=<my_caroot.pem_file_path> \ <1>
+  --from-file=user.crt=<my_cert.pem_file_path> \
+  --from-file=user.key=<my_key.pem_file_path>
+----
+<1> The `ca.crt` can be omitted to use the system's root CA set. For example, if you are using a public cloud managed Kafka service such as Red Hat OpenShift Streams for Apache Kafka.
+
+. Create or modify a `KafkaSink` object and add a reference to your secret in the `auth` spec:
++
+[source,yaml]
+----
+apiVersion: eventing.knative.dev/v1alpha1
+kind: KafkaSink
+metadata:
+   name: <sink_name>
+   namespace: default
```

Comment:
for consistency of the doc and the original sentence "Create the certificate files as a secret in the same namespace as your `KafkaSink` object:"
```suggestion
   namespace: <namespace>
```